### DISCUSSION TITLE
Remove white border?

### DISCUSSION THREAD

**Discussion by PHLAK** [ddterm/gnome-shell-extension-ddterm#82] [Q&A] [answered]
Is there a way to remove the 1px white-ish border around the terminal? If not, can we get an option to disable this?

<p align="center">
    <img src="https://user-images.githubusercontent.com/53531/134555829-c4acf759-94df-421d-a3ec-d1544dc98bf9.png" alt="terminal-border" width="50%">
</p>

**Comment by amezin** [comment, marked as answer]
Option added in https://github.com/amezin/gnome-shell-extension-ddterm/commit/1861432b6bcef6afb0caf2282499225489bbf34d

**Reply by PHLAK** [reply]
Awesome, thank you!
